src/shutdown-handler.ts

```diff
@@ -19,7 +19,7 @@ async function startShutdown() {
     return;
   }
   isShuttingDown = true;
-  const timeoutMs = 5000;
+  const timeoutMs = parseInt(process.env['STACKS_SHUTDOWN_FORCE_KILL_TIMEOUT'] ?? '60') * 1000;
   let errorEncountered = false;
   for (const config of shutdownConfigs) {
     try {
```

Comment:
Potentially related to shutdown problems -- I recently came across line https://github.com/hirosystems/stacks-blockchain-api/blob/ebd8fff7f383cdd6a1c354fb362b9f7b3bb2efc0/src/helpers.ts#L708-L711

AFAIK this can lead to unhandledPromiseRejection errors, see https://eslint.org/docs/latest/rules/no-async-promise-executor

I think it needs to catch and reject the promise, like:
```ts
    new Promise(async (resolve, reject) => {
      try {
        await promise;
      } catch (error) {
        reject(error);
      } finally {
        clearTimeout(timer);
        resolve(true);
      }
    }),
```

Reply:
Great catch, I'll add the fix to this PR

Reply:
Done. Moved the executor to *not* be async following the docs you linked to.